submissions/5mountains/dom-api-task/js/main.js

```diff
@@ -0,0 +1,118 @@
+async function fetchAsync () {
+    let response = await fetch('./data/data.json');
+    let data = await response.json();
+    return data;
+}
+  
+fetchAsync()
+    .then(data => {
+        let details = createHtml(data);
+        drawPage(details);
+        toggleTabs();
+        animation();
+    })
+    .catch(reason => console.log(reason.message));
+
+function createHtml(data) {
+    const menuItems = document.createDocumentFragment();
+    const heroItems = document.createDocumentFragment();
+    data.pancakes.forEach((elem, index) => {
+        //nav
+        //nav item (li)
```

Comment:
What the meaning of comment? :) Looks like redundant for me

Reply:
My bad! I forgot to remove all comments and console.log in this folder))) I'll fix it!

Reply:
done